nilearn/datasets/atlas.py

```diff
@@ -889,10 +976,25 @@ def fetch_atlas_aal(version='SPM12', data_dir=None, url=None, resume=True,
     see :footcite:`AAL_atlas`,
     and :footcite:`TZOURIOMAZOYER2002273`.
 
+    .. warning::
+
+        The maps image (``data.maps``) contains 117 unique integer values
+        defining the parcellation. However, these values are not consecutive
+        integers from 0 to 116 as is usually the case in Nilearn.
+        Therefore, these values shouldn't be interpreted as indices for the
+        list of label names.
+        For example, to get the name of the region corresponding to label
+        5021 in the image, you should do:
+
+        .. code-block:: python
+
+            # This should print 'Lingual_L'
+            data.labels[data.indices.index('5021')]
```

Comment:
I would maybe highlight the fact that the region ids are provided as strings in `indices` but are ints in the image. and could we add an example going in the other direction (find the value to look for in the map if we want region `"Precentral_L"`?